docs/src/explanations/explanations.md

```diff
@@ -28,7 +28,6 @@ read these documents in the following order:
 * [Memories](memories)
 * [Interfaces and Connections](interfaces-and-connections)
 * [Black Boxes](blackboxes)
-* [Intrinsics](intrinsics)
```

Comment:
@darthscsi i moved this down to be just above annotations since I think it's a rather advanced topic, WDYT